x-pack/test/security_solution_endpoint/apps/endpoint/policy_details.ts

```diff
@@ -186,6 +196,11 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
               type: 'elasticsearch',
             },
           },
+          fleet: {
```

Comment:
These look fine to me. @paul-tavares do we need to make changes in the UI code that creates the policy?

Reply:
@jonathan-buttner no, I don't think this impacts the Endpoint Integration Policy because its outside of the Integration `input` - that's the only one we manipulate in the Endpoint Policy Details page.